backend/src/routes/api/service/pipelines/index.ts

```diff
@@ -0,0 +1,24 @@
+import { DSPipelineKind } from '../../../../types';
+import { proxyService } from '../../../../utils/proxy';
+
+export default proxyService<DSPipelineKind>(
+  {
+    apiGroup: 'datasciencepipelinesapplications.opendatahub.io',
+    apiVersion: 'v1alpha1',
+    kind: 'DataSciencepipelinesApplication',
+    plural: 'datasciencepipelinesapplications',
+  },
+  {
+    port: 8888,
+    prefix: 'ds-pipeline-',
+  },
+  {
+    // Use port forwarding for local development:
+    // kubectl port-forward -n <namespace> svc/ds-pipeline-pipelines-definition 8888:8888
```

Comment:
Head of code uses port `8443` why is this different?

Reply:
Because the DSPA created for v1 pipelines doesn't support TLS, this is the only working port exposed by the service on ODH 2.8.x.

Reply:
I was just about to post a screenshot of this... it's possible I'm going mad here - there _is_ an oath port exposed on the service at port 8443?

I'll change this now.